webui/src/App.js

```diff
@@ -21,21 +21,40 @@ import {
   Link
 } from "react-router-dom";
 
+// Amount of panels.
+let panels = 5;
+let panel_indexes = init_array(panels, true);
```

Comment:
This doesn't need to be global and can be next to the ValueMonitor stuff directly.

Reply:
You might not even need this variable if you do

```
{[...Array(num_panels).keys()].map(i => (
   <ValueMonitor index={i}/>)
)}
```

below